modules/learn/pages/clusters-and-availability/xdcr-with-scopes-and-collections.adoc

```diff
@@ -98,6 +102,129 @@ However, as an alternative to an explicit mapping between `ScopeB` and `ScopeY`,
+
+[#rules-for-explicit-mappings]
+== Rules for Explicit Mappings
+
+Explicit mappings are established by means of _rules_.
+Each rule _affirms_ or _denies_ that replication should occur between a source scope or collection and a target scope and collection.
+When multiple rules are specified in the establishing of a single replication, the rules are applied with a fixed order of priority.
+This order is represented in the table below: a rule with a lower priority-number takes higher priority.
+
+[cols="3,6,9,6"]
+|===
+| Priority | Rule | Description | Syntax
+
+| 0
+| _scope.collection_ to _scope.collection_ affirmation
+| Maps a single source _scope.collection_ to a single target _scope.collection_, and affirms that replication should proceed between them.
+| `{"source_scope.source_collection":"target_scope.target_collection"}`
+
+| 1
+| _scope.collection_ denial
+| Specifies that a single source _scope.collection_ should not be replicated.
+| `{"source_scope.source_collection":null}`
+
+| 2
+| _scope_ to _scope_ affirmation
+| Maps a single source _scope_ to a single target _scope_, and affirms that replication should proceed between them.
+| `{"source_scope":"target_scope"}`
+
+| 3
+| _scope_ denial
+| Specifies that a single source _scope_ should not be replicated.
+| `{"source":null}`
+
+|===
+
+Additional information on each of these rules is provided below.
+
+[#priority-0]
+=== Priority 0
+
+A single, unique collection under a single unique scope on the source is mapped to a single, unique collection under a single unique scope on the target, and is affirmed for replication.
+For example, the expression `{"inventory.airport":"MyInventory.MyAirport"}` affirms that the collection `airport`, within the source-scope `inventory`, should be replicated to the collection `MyAirport`, within the target-scope `MyInventory`.
+
+A source collection is not permitted to be mapped (by means of multiple rules) to multiple target collections.
+For example, the expression `{"inventory.airport":"MyInventory.MyAirport","inventory.airport":"MyInventory.airport"}` generates an error.
+
+However, multiple source collections can be mapped (by means of multiple rules) to a single target collection.
+For example, the expression `{"inventory.airport":"MyInventory.MyAirport", "inventory.MyAirport":"MyInventory.MyAirport"}` is valid, and affirms replication from the collections `airport` and `MyAirport`, located within the source-scope `inventory`, to the collection `MyAirport`, located within the target-scope `MyInventory`.
+
+If the replication of multiple source-collections to a single target collection results in conflicts, the procedure for xref:learn:clusters-and-availability/xdcr-conflict-resolution.adoc[XDCR Conflict Resolution] that has been established for the bucket is applied.
```

Comment:
Once MB-44932 is fixed, we won't need this section.

Reply:
I've removed the paragraph about conflict resolution.